examples/rainmaker_example/main/CMakeLists.txt

```diff
@@ -1,2 +1,10 @@
+set(requires esp_rainmaker nvs_flash hts221 ssd1306 json_parser app_wifi app_reset esp32_azure_iot_kit)
```

Comment:
Given that hts221 and ssd1306 are public dependencies of esp32_azure_iot_kit, is it necessary to add them explicitly?

Reply:
They aren't there in CMakeLists.txt. I added only these dependencies, which was missing, when I tried build.